lib/coverband/reporters/web.rb

```diff
@@ -126,7 +140,7 @@ def clear
         else
           notice = "web_enable_clear isn't enabled in your configuration"
         end
-        [301, {"Location" => "#{base_path}?notice=#{notice}"}, []]
```

Comment:
note these should have always been 302s, just realized it while building this out